README.md

```diff
@@ -19,9 +19,9 @@ Hopefully, it will get better soon.
     * It should look like:
     ```
     [github]
-       token = "my_token"
+       token = my_token
```

Comment:
it works with the quotes. that's what i have in my file

Reply:
I think it doesn't work, that's why you need the `GITHUB_TOKEN` env var